static/docs/use-cases/data-storage-on-external-drive.md

```diff
@@ -0,0 +1,218 @@
+# Data Storage on External Drive
+
+Sometimes the data may be stored on an
+[external hard drive](https://whatis.techtarget.com/definition/external-hard-drive).
+Usually such data are huge, which means that they won't fit on our home
+directory, and even if they did, it would certainly take a long time to copy
+them back and forth from the external drive to the internal one. For example
+let's say that the size of the external drive is 16TB, while the hard drive of
+our home directory is only 320GB.
+
+In this case we would like to process the data where they are located (on the
+external drive). We also would like to save the results there, and certainly to
+store the <abbr>cached</abbr> files there as well.
+
+The most easy way to do this would be to locate the <abbr>workspace</abbr> on
+the external drive itself. If we assume that the external drive is mounted on
+`/mnt/data/`, then it could be done like this:
+
+```dvc
+$ sudo su
+# cd /mnt/data/
```

Comment:
I'm not sure I understand the logic behind this code snippet. Could you elaborate a little bit on what are you trying to say here, please?

Reply:
The context is this:

-----

The most easy way to do this would be to locate the <abbr>workspace</abbr> on
the external drive itself. If we assume that the external drive is mounted on
`/mnt/data/`, then it could be done like this:

```dvc
$ sudo su
# cd /mnt/data/
# git init
# dvc init
```

-----

> If we assume that the external drive is mounted on `/mnt/data/`

This means something like this: `mkdir -p /mnt/data ; mount /dev/sdb1 /mnt/data`. Usually USB disk drives are mounted automatically on a path that is a bit more complicated, which may look like this: `/media/<user>/<disk-label>`, but for simplicity and for generalization we may assume that it is `/mnt/data`.

Now, `/mnt/data/` is the directory where the data files are located, for example `/mnt/data/raw/`. Instead of initializing a workspace on our home directory and moving the data there, we initialize the workspace on the data directory itself.

Reply:
Then the name should be different. It's confusing to have a project in the data directory. It'll look like this most likely: `/mnt/my-project/data` and `/mnt/my-project` will be initialized as a Git repo.

Otherwise I don't understand where will you put your code? Into `/mnt/data` directly?

Reply:
> Otherwise I don't understand where will you put your code? Into `/mnt/data` directly?

Yes, that is the assumption.

> Then the name should be different.

Would it be better if `/mnt/data` is renamed to `/mnt/external-drive`?

Reply:
@dashohoxha no. I just haven't see anyone using `data` root directory as a workspace. Usually people tend to name their projects with some meaningful names - `iterrative/example-get-started`. Would be strange to have a git repo named `data` with some NLP code inside. Or may be I'm still missing your point.

Reply:
The idea is to initialize the workspace at the same place where the data are located.
Maybe you are missing my point, I am not sure. But I have run out of explanations.
What do you suggest?